wandb/sdk/internal/file_stream.py

```diff
@@ -335,6 +335,7 @@ def __init__(
         self._run_id = run_id
         self._start_time = start_time
         self._client = requests.Session()
+        timeout = timeout or 0
```

Comment:
how is this happening? timeout is default to 0. weird

Reply:
ok based on the stack trace it is `self._settings._file_stream_timeout_seconds`
let's see why it's none

Reply:
we set the default value to 60... really strange :(

Reply:
it must be coming from a place where we just use SettingsStatic and not start with the regular settings and then convert them to such. Don't feel like tracing it down as this code path is reworked in nexus.